src/main.cc

```diff
@@ -324,6 +324,10 @@ int main(int argc, char* argv[]) {
       srv->Stop();
     }
   };
+
+  // Generate new replication id if not a replica
+  if (!srv->IsSlave()) storage.ShiftReplId();
+
```

Comment:
Master was added after starting the server, so it should always be true here.

Reply:
you are right, i missed this, we set master info in `Server::Start ` instead of loading config file.